src/styles/fonts.scss

```diff
@@ -56,7 +56,7 @@
   src: url("/fonts/ProximaVara/proxima_vara_regular.woff2"),
     url("/fonts/ProximaVara/proxima_vara_regular.woff");
   font-weight: normal;
-  font-display: swap;
+  font-display: optional;
```

Comment:
confirm that optional is optimal for us

Reply:
`optional` is an optimization that prevents Layout shift. So basically there would be a window (100ms) for Proxima font to load. During this window the text doesn't show and if the font finishes loading within this window, the text shows in Proxima font, if not, it will show the fallback font https://web.dev/preload-optional-fonts/